src-electron/util/env.js

```diff
@@ -73,14 +73,25 @@ export function builtinSilabsZclGeneralXmlFile() {
 }
 
 /**
- * Used to retrive general-special.xml by zcl reload test in zcl-loader.test.js
+ * Used to retrieve general-special.xml by zcl reload test in zcl-loader.test.js
  *
  * @returns path to general-special.xml file used by zcl-loader.test.js
  */
 export function builtinSilabsSpecialZclGeneralSpecialXmlFile() {
   return locateProjectResource('./zcl-builtin/silabs/general-special.xml')
 }
 
+/**
+ * Used to retrieve cluster-to-component-dependencies.json by zcl reload test in zcl-loader.test.js
+ *
+ * @returns path to cluster-to-component-dependencies.json file used by zcl-loader.test.js
+ */
+export function builtinZclExtensionClusterToComponentFile() {
+  return locateProjectResource(
+    './test/gen-template/zigbee2/cluster-to-component-dependencies.json'
```

Comment:
env.js is not test file. But it refers to a test file here. You should put this in `test-util`, not env.js

Here we keep "zcl-builtin" stuff, but `test` stuff is not even packaged.

Reply:
Done